appinsights/nodejs.go

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2018-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package appinsights
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/buildpacks/libcnb"
+
+	"github.com/paketo-buildpacks/libpak"
+	"github.com/paketo-buildpacks/libpak/bard"
+	"github.com/paketo-buildpacks/libpak/effect"
+	"github.com/paketo-buildpacks/libpak/sherpa"
+	"github.com/paketo-buildpacks/microsoft-azure/internal/common"
+	"github.com/paketo-buildpacks/microsoft-azure/internal/nodejs"
+)
+
+const (
+	NodeModule = "applicationinsights"
+	NodePath   = "BPI_AZURE_APPLICATION_INSIGHTS_NODE_PATH"
```

Comment:
```suggestion
	AppInsightsNodePath   = "BPI_AZURE_APPLICATION_INSIGHTS_NODE_PATH"
```
I suggest making this more specific only because, when I saw the variable used below, I assumed we were setting a default for `NODE_PATH` and was momentarily confused.